### ISSUE TITLE
Dataset cant' be reachable anymore

### ISSUE THREAD

**Issue by asahi417** [neptune-ai/open-solution-mapping-challenge#226]
Hi, can anyone share mapping-challenge dataset? CrowdAI was closed and they don't accept new users anymore on their platform. I know CrowdAI becomes AICrowd and they have a corresponding page (https://www.aicrowd.com/challenges/mapping-challenge) but the dataset is still depending on CrowdAI platform and it doesn't allow AICrowd users to utilize it...

**Comment by jakubczakon** [comment]
I was told by the organizer that the dataset has been moved here:
https://www.aicrowd.com/challenges/mapping-challenge (as you said).
If you register to aicrowd you should be able to download it.

**Comment by asahi417** [comment]
Right, I found the dataset there, but it redirects to crowdAI sign-in page (https://www.crowdai.org/participants/sign_in), and the issue is that my AIcrowd account can't be verified by crowdAI sign-up system, so I assume AIcrowd doesn't have anything to do with crowdAI platform.

**Comment by bractus** [comment]
Registrations are closed, can someone please send me the data?

**Comment by salmagro** [comment]
I have found partial data here: https://www.kaggle.com/kmader/synthetic-word-ocr
